specification/2.0/README.md

```diff
@@ -265,6 +265,8 @@ The [node transformations](#transformations) and [animation channel paths](#anim
 
 The glTF asset contains zero or more *scenes*, the set of visual objects to render. Scenes are defined in a `scenes` array. An additional property, `scene` (note singular), identifies which of the scenes in the array is to be displayed at load time.
 
+All nodes listed in `scene.nodes` array must be the roots of trees in the node forest (see the next section for details).
```

Comment:
I love the term `node forest` but in the next section you redeclare it as `node hierarchy`.  Perhaps pick one or the other?

Reply:
It's not me. "node hierarchy" is used many times in the spec, while "forest" is quite stable term w.r.t. graph theory. Do you think we can introduce these notions better?

Reply:
This just jumped out at me from reading this set of diffs.  If no one is actually bothered by it, I'm OK.

Reply:
I'm not bothered by "node forest" but I think its usage is unnecessary here. How about this?
> All nodes listed in `scene.nodes` array must be root nodes (see the next section for details).